.depcheckrc.json

```diff
@@ -7,6 +7,7 @@
     "@yarnpkg/types",
     "prettier-plugin-packagejson",
     "ts-node",
-    "typedoc"
+    "typedoc",
+    "buffer"
```

Comment:
eslint does not like that I'm overshadowing node.js's buffer, but it is not available in the browser so I need a poyfill